corsy.py

```diff
@@ -29,6 +29,7 @@
 parser.add_argument('-t', help='thread count', dest='threads', type=int, default=2)
 parser.add_argument('-d', help='request delay', dest='delay', type=float, default=0)
 parser.add_argument('-q', help='don\'t print help tips', dest='quiet', action='store_true')
+parser.add_argument('-f', help='import from burp logs', dest='file')
```

Comment:
1. This argument doesn't accept any additional value, hence add `action='store_true'` to make it Boolean.
2. I think `-b` is a better name for it and the `dest` value can be `burp_file` or `burp_log`.
4. The `help` text should be `treat input file as Burp Suite logs`
3. Please change the position of this line according to it's length.